src/software/ai/hl/stp/stp.h

```diff
@@ -160,4 +160,6 @@ class STP
     std::unique_ptr<FSM<PlaySelectionFSM>> fsm;
     bool override_play_changed;
     std::unique_ptr<Play> override_play;
+    // inter play communication
+    InterPlayCommunication inter_play_communication;
```

Comment:
so this means that stp will hold the message, as we transition between different plays?

Reply:
yes